python/llm/example/GPU/QLoRA-FineTuning/alpaca-qlora/README.md

```diff
@@ -0,0 +1,49 @@
+# Alpaca QLoRA Finetuning (experimental support)
+
+This example ports [Alpaca-LoRA](https://github.com/tloen/alpaca-lora/tree/main) to BigDL-LLM QLoRA on [Intel GPUs](../../README.md).
+
+### 0. Requirements
+To run this example with BigDL-LLM on Intel GPUs, we have some recommended requirements for your machine, please refer to [here](../../README.md#requirements) for more information.
+
+### 1. Install
+
+```bash
+conda create -n llm python=3.9
+conda activate llm
+# below command will install intel_extension_for_pytorch==2.0.110+xpu as default
+# you can install specific ipex/torch version for your need
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+pip install transformers==4.34.0
+pip install fire datasets peft==0.5.0
```

Comment:
add `pip install accelerate==0.23.0` ?

Reply:
> add `pip install accelerate==0.23.0` ?

Following the provided commands, `accelerate==0.21.0` is installed and experiments work fine. Is it necessary?